packages/search/app/styles/navi-search.less

```diff
@@ -0,0 +1,156 @@
+@font-face {
+  font-family: 'denali-icon-font';
+  src: url('denali-icon-font.eot?1576615905868');
+  src: url('denali-icon-font.eot?#iefix-1576615905868') format('embedded-opentype'),
+    url('denali-icon-font.svg?1576615905868') format('svg'), url('denali-icon-font.woff?1576615905868') format('woff'),
+    url('denali-icon-font.ttf?1576615905868') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+.d-icon[class^='d-'],
+.d-icon[class*=' d-'] {
+  font-family: 'denali-icon-font' !important;
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  font-size: 24px;
+  vertical-align: middle;
+  text-decoration: inherit;
+  text-rendering: optimizeLegibility;
+  /* Better Font Rendering */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+/* Glyphs list */
+
+.d-dashboard-tile:before {
+  content: '\eb4c';
+}
+
+.d-dashboard:before {
+  content: '\eb4d';
+}
+
+.d-file-search:before {
+  content: '\ebab';
+}
+
+.d-file-text:before {
+  content: '\ebad';
+}
+
+.d-search:before {
+  content: '\ecfd';
+}
+
+.d-table:before {
+  content: '\ed5f';
+}
+
+.d-text-center:before {
+  content: '\ed6b';
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+td {
+  padding: 0;
+  text-align: left;
+}
+
+.navi-search-result {
+  font-family: Helvetica, 'Arial', sans-serif;
+  font-weight: 400;
+  line-height: 1.5;
+  background: @denali-grey-100;
+  color: @denali-grey-800;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+}
+
+.d-icon {
+  font-size: 2.4rem;
+}
+
+a {
+  color: @denali-brand-600;
+  text-decoration: none;
+  font-size: inherit;
+}
+
+a:hover {
+  color: @denali-brand-700 !important;
+}
+
+a[disabled] {
+  color: @denali-grey-800;
+  cursor: not-allowed;
+}
+
+table {
+  border-collapse: collapse;
+  width: 100%;
+  text-align: left;
+}
+
+table > tbody > tr > td {
```

Comment:
will these styles affect tables outside of the search?

Reply:
I think they shouldn't because we import the navi-search.less only in search

Reply:
but if I happen to pull in both navi-search and navi-reports into my app? will there be conflict?

Reply:
I think it wouldn't pick up my styles, you'd have to import the navi-search.less file. Same as now I had to import the navi-core.less to get core stylings

Reply:
We should use classes instead of styling on element types.